lib/ansible/modules/network/basics/uri.py

```diff
@@ -411,8 +411,12 @@ def main():
 
     # Grab all the http headers. Need this hack since passing multi-values is
     # currently a bit ugly. (e.g. headers='{"Content-Type":"application/json"}')
+    deprecations = set()
     for key, value in six.iteritems(module.params):
         if key.startswith("HEADER_"):
+            deprecations.add('Supplying headers via HEADER_* is deprecated and '
```

Comment:
It does, just not there.  Due to the fact that we are looping all `HEADER_` keys, I am placing all of the deprecations into a `set`, so that they aren't duplicated, and then later, adding them to `module.deprecate` on lines 468-469